tmt-app/src/Components/MajorPage.jsx

```diff
@@ -103,6 +120,52 @@ function MajorPage() {
             </Grid>
           );
         })}
+        <Grid item xs={12} md={6}>
+          <Card
+            sx={{
+              boxShadow: "none",
+              backgroundColor: "Avacado.main",
+              height: "100%",
+              boxSizing: "border-box",
+            }}
+          >
+            <Typography variant="h3" sx={{ paddingBottom: "1.5rem" }}>
+              Any extra resources?
+            </Typography>
+            <Typography variant="body">
+              <List>
+                {resourcesLink.map((resource, index) => {
+                  return (
+                    <Container
+                      key={`${resource.name}-${index}`}
+                      sx={{
+                        display: "flex",
+                        alignItems: "center",
+                        margin: "0.75rem 0",
+                      }}
+                    >
+                      <ListItemIcon>
+                        <img src={BobaIcon} alt="Boba Icon" />
+                      </ListItemIcon>
+                      <Link
+                        href={resource.link}
+                        rel="noopener noreferrer"
+                        target="_blank"
+                        sx={{
+                          color: "black",
+                          cursor: "pointer",
```

Comment:
Doesn't this style come default with links? Did we override it somewhere that requires this?

Reply:
I just changed it right now but I left in the underline cause the default one is different.